### DISCUSSION TITLE
Beets appears to silently 'hang'

### DISCUSSION THREAD

**Discussion by simonmcnair** [beetbox/beets#4700] [General]
This may be as a result of using -vvv but It appears the messages from the multiprocess import can hide the prompt for input when 'ask' is chosen in quiet_fallback and .  This results in the prompt disappearing from the screen and then nothing appears to happen any more.  No log messages, no sign of activity at all.

Would this be considered a bug if it's an edge case where debug is enabled ?  I suspect this could happen with any degree of logging.  Arguably when the ask prompt appears all other processes should pause until it is answered ?

Just curious if anyone else has come across this ?

I think these are the settings that cause it to happen, executed as /usr/local/bin/beet -vvv import .

extract from config.yaml

```import:
    write: yes
    copy: no
    move: yes
    resume: yes
    incremental: no
    quiet_fallback: ask
    timid: no
    quiet: no
    none_rec_action: skip
    default_action: apply
    duplicate_action: keep
    group_albums: yes
```

**Comment by simonmcnair** [comment]
I wonder if this is a result of using quiet: no and not answering any questions for a while, and each one holds open multiple files ?

**Comment by simonmcnair** [comment]
[beet.log](https://github.com/beetbox/beets/files/10941345/beet.log)

I don't know if this helps or not.

**Comment by DjSlash** [comment]
You have `quiet: no` and also don't provide `-q` with your import command, so `quiet_fallback` should not take effect.  It is known that beets may [seem to hang after import finishes](https://beets.readthedocs.io/en/stable/faq.html#seem-to-hang-after-an-import-finishes). 

Quote from that link:
> Beets uses a multithreaded importer that overlaps many different activities: it can prompt you for decisions while, in the background, it talks to MusicBrainz and copies files. This means that, even after you make your last decision, there may be a backlog of files to be copied into place and tags to be written. (Plugin tasks, like looking up lyrics and genres, also run at this time.) If beets pauses after you see all the albums go by, have patience.

**Comment by simonmcnair** [comment]
This was definitely not a hang in retrospect.  It stops and then crashes when you press a key.  Previously I did ctrl+c but it exits even with any keyboard input.

**Comment by simonmcnair** [comment]
I have opened an issue on this with the error https://github.com/beetbox/beets/issues/4701
